primitives/src/constants.rs

```diff
@@ -29,6 +29,7 @@ use crate::types::{Balance, BlockNumber};
 use frame_support::{parameter_types, PalletId};
 
 // Definitions for time
+pub const BLOCKS_PER_YEAR: BlockNumber = (BLOCKS_PER_DAY * 36525) / 100;
```

Comment:
Not able to follow the math here, shouldn't it be just BLOCKS_PER_DAY * 365 ? Or there is something I don't see.